cmd/cerbosctl/audit/audit.go

```diff
@@ -145,16 +145,20 @@ func streamLogsToWriter(writer auditLogWriter, entries <-chan *client.AuditLogEn
 		if err != nil {
 			return fmt.Errorf("error while receiving access logs: %w", err)
 		}
-		if err := writer.write(aLog); err != nil {
-			return err
+		if aLog != nil {
+			if err := writer.write(aLog); err != nil {
+				return err
+			}
```

Comment:
Maybe add a `continue` here because we know that if access log is not nil, decision log is definitely nil.